src/js/theme.js

```diff
@@ -516,10 +516,11 @@ class Theme {
     initMermaid() {
         const $mermaidElements = document.getElementsByClassName('mermaid');
         if ($mermaidElements.length) {
-            mermaid.initialize({startOnLoad: false, theme: 'null'});
+            mermaid.initialize({startOnLoad: false, theme: 'default'});
             this.util.forEach($mermaidElements, $mermaid => {
                 mermaid.mermaidAPI.render('svg-' + $mermaid.id, this.data[$mermaid.id], svgCode => {
                     $mermaid.insertAdjacentHTML('afterbegin', svgCode);
+                document.getElementById('svg-' + $mermaid.id).children[0].remove();
```

Comment:
To remove these default style, a dirty work is done to remove them from the DOM. I am not quite happy with this implementation, but this is the only solution for the latest mermaid. Hopefully they can fix their bugs very soon.